src/ServiceInsight/SequenceDiagram/ModelCreator.cs

```diff
@@ -25,6 +26,7 @@ public ModelCreator(List<StoredMessage> messages, IMessageCommandContainer conta
             this.container = container;
 
             Initialize();
+            OrderHandlers();
```

Comment:
The main thing is that `Initialize` is doing a few things, including a first-step ordering. My thinking was that doing it as a part of Initialize (also a generalized name) would not making the second-step ordering obvious enough.